tools/test_api.py

```diff
@@ -17,6 +17,7 @@
 Author: Przemyslaw Wirkus <Przemyslaw.wirkus@arm.com>
 """
 from __future__ import print_function
+import six
```

Comment:
`six` is [not currently a required package from mbed-os](https://github.com/ARMmbed/mbed-os/blob/master/requirements.txt), so we can't use this without adding it as a requirement.

But see my other suggestion below.

Reply:
Huh. I was under the impression that six was already required due to https://github.com/ARMmbed/mbed-os/pull/5022

Reply:
My guess is that a different package ends up installing it as a dependency. I noticed it showed up after a fresh virtual environment.